src/schema/notifications.ts

```diff
@@ -485,7 +485,6 @@ export class ZimbraNotifications {
 			itemsToWrite = data.getNewMail;
 		} catch (exception) {
 			itemsToWrite = [];
-			console.error(exception);
```

Comment:
It's known behaviour so not required to console it. (Change suggested in PREAPPS-1860)
cc: @aayushvijayvergiya @itsmeprashant